requirements.txt

```diff
@@ -8,4 +8,5 @@ lxml
 wellcomeml[deep-learning]==2020.11.1
 docutils==0.15.2
 bokeh==2.1.0
-scholarly
\ No newline at end of file
+scholarly
```

Comment:
that is an interesting package https://github.com/scholarly-python-package/scholarly, check @aCampello

Reply:
Nice!